load.bzl

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def repositories():
+    if not native.existing_rule("bazel_skylib"):
```

Comment:
For my own understanding: does this pattern mean if `WORKSPACE` has already loaded these rules we won't load a different version?

https://docs.bazel.build/versions/master/skylark/lib/native.html#existing_rule
> Note: If possible, avoid using this function. It makes BUILD files brittle and order-dependent.

😬 

and yet this pattern seems common? https://chromium.googlesource.com/external/github.com/google/protobuf/+/HEAD/protobuf_deps.bzl

Reply:
Yeah I am just trying to follow standard practice?

https://github.com/bazelbuild/rules_go/blob/master/go/private/repositories.bzl#L44-L47

https://github.com/bazelbuild/bazel-gazelle/blob/master/deps.bzl#L89-L95

Reply:
ACK just not aware of / certain of the practice :sweat_smile: